docs/wiki/[Examples]-Create-and-Assign-Custom-RBAC-Roles.md

```diff
@@ -103,6 +103,8 @@ If you don't already have an `archetype_extension_es_landing_zones.tmpl.json` fi
 This code will assign your new `Reader-Support-Tickets` role to a group named `Contoso Reader and Support Tickets`.
 In order to assign the `Reader-Support-Tickets` role to the group, you need to use the groups objectID which can be located in Azure Active Directory.
 
+Instead of group, if you would like to assign the role to a SPN in the form of application registered in Azure AD then make sure to use the Object ID of the Service Principal related to that application which you can find from the "Enterprise applications" blade of Azure AD Portal. For more details on the relation between application and service principal objects please see [this article](https://learn.microsoft.com/en-us/azure/active-directory/develop/app-objects-and-service-principals).
```

Comment:
Please can you take the locale out of the URL?

Reply:
Thanks put for the feedback, updated the PR accordingly